signup.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html><head>
+  <style>
+  button {
+      background-color: #4CAF50;
+      color: white;
+      margin: 8px 0;
+      border: none;
+      cursor: pointer;
+      padding: 7px 40px;
+      width: 15%;
+  }
+  
+  button:hover {
+      opacity: 0.8;
+  }
+  body {
+    background-color: #BDB76B;
+}</style>
+    
+
+<title>Maintainance Tracker</title>
+</head>
+<body>
```

Comment:
Can you try formatting your code.
If you are using vscode on windows use `alt + shift + F` or `option + shift + F` on a MAC.
This is done automatically on vscode. You don't need to install any extension

Reply:
thanks. this seems easy